include/jemalloc/internal/tcache_inlines.h

```diff
@@ -50,6 +50,22 @@ tcache_bin_settings_backup(tcache_t *tcache,
 	}
 }
 
+static inline cache_bin_sz_t
+tcache_get_default_ncached_max(szind_t ind) {
+	assert(ind < TCACHE_NBINS_MAX);
+	return opt_tcache_ncached_max[ind].ncached_max;
+}
+
+static inline bool
+tcache_get_default_ncached_max_set(szind_t ind) {
+	return opt_tcache_ncached_max_set[ind];
+}
+
+static inline const cache_bin_info_t *
+tcache_get_default_ncached_max_arr(void) {
```

Comment:
To be symmetrical, add getters for both `opt_tcache_ncached_max` and `opt_tcache_ncached_max_set` (although they are only used once for now) to warn people that they should not be set besides initialization. 

Also add an array getter for `opt_tcache_ncached_max` because the whole array as an input makes the auto tcache initialization unified with `tsd_tcache_data_init_with_bin_settings`. To further highlight the array should not be changed, add `const` to the getter return value and also corresponding functions' parameters.

Reply:
> To be symmetrical, add getters for both opt_tcache_ncached_max and opt_tcache_ncached_max_set (although they are only used once for now) to warn people that they should not be set besides initialization.

See the comments I just added. Providing the getters kind of implies "they are fine to use when you want". For data used only during init and not after like `opt_tcache_ncached_max_set`, we should try making it static and limit its usage, and not having the getter.